### PR TITLE
Add 'npm start' and 'npm test' scripts

### PR DESCRIPTION
**Pull request by kmees**
For people who don't like to search for gulp task names or don't have a
global gulp installation.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,10 @@
     "url": "https://github.com/OfficeDev/office-ui-fabric-react"
   },
   "license": "MIT",
+  "scripts": {
+    "start": "gulp serve",
+    "test": "gulp test"
```
Comment (dzearing) [inline package.json]: minor nit , we could also add this:

"build": "gulp --production"

**Comment by dzearing** [comment]
Approved

File: package.json (hunk 1 above)
Comment (dzearing) [inline package.json]: I can add as a separate pr.

**Comment by dzearing** [comment]
Thanks! Was meaning to add this.

File: package.json (hunk 1 above)
Comment (kmees) [inline package.json]: I just wanted to support the 'standard' scripts for now

**Pull request merged**